Automattic/newspack-plugin
# feat: segments criteria ui

### All Submissions:






### Changes proposed in this Pull Request:



1204189396255901-as-1204857763418497

Changes the Segments UI to use the new criteria system and updated schema.

### How to test the changes in this Pull Request:

Confirm updated tests are passing and have appropriate coverage.

1. Checkout this branch and https://github.com/Automattic/newspack-popups/tree/epic/rearchitecture
2. Create, edit and delete segments and confirm everything works as expected
3. Create a segment using all available criteria
4. Visit the "Segments" home and confirm the description renders correctly for every item
5. Visit the "Campaigns" home and confirm the same description also renders correctly

### Other information:
